### PR TITLE
Restore node 4 compatibility and remove node 0.12 from travis

### PR DESCRIPTION
**Pull request by adrianheine**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -428,4 +438,4 @@ function plugin(options, Parser) {
       }
     }
   };
-};
+}
```
Comment (RReverser) [inline index.js]: Can you please return this semicolon?
```suggestion
};
```

Comment (adrianheine) [reply index.js]: If you want me to, sure, but that's a function statement, right?

**Comment by adrianheine** [comment]
I try to support node 4 since it is still in Debian stable and not much trouble.

File: index.js (hunk 1 above)
Comment (RReverser) [reply index.js]: Oh, I thought it was the end of `module.exports`. You're right.

**Comment by RReverser** [comment]
SGTM, published as 5.0.1.

**Pull request merged**